macro/src/codegen.rs

```diff
@@ -189,6 +188,22 @@ impl SpannedClassInfo {
             .collect()
     }
 
+    // XX: Clearly, we'll need more sophisticated resolution of what types we descend from, but for now we can at least
+    // inject the "everything is an Object" root.
```

Comment:
I wouldn't expect it to be necessary for us to inject Object -- but I guess you are just leaving a "TODO" to implement the transitive closure?

Reply:
Yeah, meant as a TODO for the full transitive closure impl. I just needed the `Object` root everywhere to make the "inspect a Java Object" example pass